README.md

```diff
@@ -168,3 +172,4 @@ Please visit [Zivid Knowledge Base][zivid-knowledge-base-url] for general inform
 [zivid-software-installation-url]: https://support.zivid.com/latest/getting-started/software-installation.html
 [zivid-download-software-url]: https://www.zivid.com/downloads
 [zivid-software-url]: http://www.zivid.com/software
+[zivid-python-releases-url]: https://github.com/zivid/zivid-python/releases
```

Comment:
I think it is better to refer directly PyPI since in theory GitHub and PyPI can bout out of sync, and it is what is in PyPI that matters in the end.

```suggestion
[zivid-python-releases-url]: https://pypi.org/project/zivid/#history
```